src/AppInstallerCLICore/Commands/InstallCommand.cpp

```diff
@@ -137,7 +138,20 @@ namespace AppInstaller::CLI
                     Workflow::OpenCompositeSource(Repository::PredefinedSource::Installed, false, Repository::CompositeSearchBehavior::AvailablePackages);
             }
 
-            context << Workflow::InstallOrUpgradeSinglePackage(false);
+            if (!context.Args.Contains(Execution::Args::Type::MultiQuery))

```

Comment:
Nit: Seems a bit odd to be using a `not` here. Would be more readable if you switched the if-else and removed the not